h/views/client.py

```diff
@@ -31,7 +31,8 @@ def _client_url(request):
 
 
 @view_config(route_name='sidebar_app',
-             renderer='h:templates/app.html.jinja2')
+             renderer='h:templates/app.html.jinja2',
+             csp_insecure_optout=True)
```

Comment:
The stream and single-page app views will still fail CSP checks because of some client-side issues. For example, annotations with LaTeX markup will try to render inline styles. Perhaps we should add this flag to the stream and single annotation views as well for the moment?